cmd/mesh/manifest-versions.go

```diff
@@ -105,8 +104,26 @@ func getVersionCompatibleMap(versionsURI string, binVersion *goversion.Version,
 		}
 	}
 	if myVersionMap == nil {
-		l.logAndFatal("This operator version ", binVersion.String(),
-			" was not found in the global manifestVersions map.")
+		return nil, fmt.Errorf("this operator version %v was not found in the global manifestVersions map", binVersion.String())
+	}
+	return myVersionMap, nil
+}
+
+func loadCompatibleMapFile(versionsURI string, l *logger) (b []byte, err error) {
```

Comment:
The logic here is not quite right. We should try versionsURI as a get if it has a http: prefix. Otherwise try it as a local file path. 
Depending on which way we try it, emit a warning about either remote URL or file.

Reply:
Yes, I fixed the logic here. Need a double-check. @ostromart

Reply:
lgtm